test/parallel/test-crypto-cipheriv-decipheriv.js

```diff
@@ -78,6 +78,78 @@ function testCipher3(key, iv) {
          `encryption/decryption with key ${key} and iv ${iv}`);
 }
 
+{
+  const Cipheriv = crypto.Cipheriv;
+  const key = '123456789012345678901234';
+  const iv = '12345678';
+
+  const instance = Cipheriv('des-ede3-cbc', key, iv);
```

Comment:
`createCipheriv`

Reply:
This line cannot use `createCipheriv` because `createCipheriv` calls constructor with new keyword always.
I want to test call constructor without new keyword.